src/components/home.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import DateFilter from './dateFilter';
+
+const Home = () => (
+  <DateFilter />
+);
```

Comment:
- [ ] as part of the requirements you should follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-\(Free-PSDs\)) including:

* Typography: font face, size, and weight
* Layout: composition and space between elements.

**_Home Page_**

Your Design           | Required Design
:-------------------------:|:-------------------------:
![image](https://github.com/gersonhiggins/capstone-financy/assets/89282221/a1b79eb9-d84d-4125-8e77-6b86eb78bf5d) | ![image](https://github.com/gersonhiggins/capstone-financy/assets/89282221/757dcd93-ac90-437c-ae1f-675ca96aa5c6)

**_Details Page_**

Your Design           | Required Design
:-------------------------:|:-------------------------:
 ![image](https://github.com/gersonhiggins/capstone-financy/assets/89282221/eb0405a1-9290-431e-87be-4f6e6eb1bd4b) | ![image](https://github.com/gersonhiggins/capstone-financy/assets/89282221/fbe723fd-e94d-4e44-9d53-02d84d7683da)